### PR TITLE
add isInfo to audits

### PR DESCRIPTION
**Pull request by kaligrafy**

### CODE REVIEW

File: packages/evolution-backend/src/models/migrations/20240205090500_addInfoAuditTypeToAuditTbl.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { Knex } from 'knex';
+
+const tableName = 'sv_audits';
+
+export async function up(knex: Knex): Promise<unknown> {
+    return knex.schema.alterTable(tableName, (table: Knex.TableBuilder) => {
+        table.boolean('is_info');
```
Comment (tahini) [inline packages/evolution-backend/src/models/migrations/20240205090500_addInfoAuditTypeToAuditTbl.ts:13]: Can an audit be both info and warning? Usually, concepts like that will use a `level` field: error, warning, info (critical, debug, trace, but we don't need those for audits I suppose)

**Review by tahini — CHANGES_REQUESTED** [review]
info/warning looks like a log's level concept. We should rather have a `level` field, with an enum or number to identify an audit level. The number allows to have a hierarchy of the levels. Otherwise, we can have an audit that can be both info and warning and that is not conceptually right.

File: packages/evolution-backend/src/models/migrations/20240205090500_addInfoAuditTypeToAuditTbl.ts (hunk 1 above)
Comment (kaligrafy) [reply packages/evolution-backend/src/models/migrations/20240205090500_addInfoAuditTypeToAuditTbl.ts:13]: Merging warning and error into a level would mean long refactoring. Let's keep as is for now, and add an issue for later?

**Comment by kaligrafy** [comment]
Replaced by https://github.com/chairemobilite/evolution/pull/372

**Pull request closed without merging**